### PR TITLE
Rename update namespace mutation 

### PR DESCRIPTION
**Pull request by NetanelBollag**
Make update namespace mutation more self explanatory, since it really just change the namespace environment

### CODE REVIEW

File: namespaces.graphql [hunk 1]
```diff
@@ -29,9 +29,9 @@ extend type Query {
...
 
 extend type Mutation {
-    """Update namespace"""
-    updateNamespace(
+    """Associate namespace to environment"""
+    associateNamespaceToEnv(
         id: ID! @validate(customConstraint: OUID),
         environmentId: ID @validate(customConstraint: OUID),
-    ): Namespace! @restApiRoute(method: PATCH, path: "/namespaces/:id", tags: ["namespaces"])
+    ): Namespace! @restApiRoute(method: PUT, path: "/namespaces/:id/environment/:environmentId", tags: ["namespaces"])
```
Comment (amitlicht) [inline namespaces.graphql:36]: I'm not sure re the REST API route here. `PUT` with the argument to update in the request URL is sort of non-standard. I'd do it: `PUT /namespace/:id/environment`, and the env id to update in the request body.

Comment (amitlicht) [reply namespaces.graphql:36]: Disclaimer: if this was discussed with @usarid then forget I said anything.

**Review by amitlicht — APPROVED** [review]
See comment

File: namespaces.graphql (hunk 1 above)
Comment (NetanelBollag) [reply namespaces.graphql:36]: No it wasn't discussed, I'll change that

**Pull request merged**